srcpkgs/onefetch/template

```diff
@@ -26,4 +27,13 @@ pre_build() {
 
 post_install() {
 	vlicense LICENSE.md
+
+	local _onefetch=${DESTDIR}/usr/bin/onefetch
+
+	vman docs/onefetch.1
+
+	for _shell in bash fish zsh; do
+		vtargetrun ${_onefetch} --generate ${_shell} > onefetch.${_shell}
+		vcompletion onefetch.${_shell} ${_shell} onefetch
```

Comment:
```suggestion
		vcompletion onefetch.${_shell} ${_shell}
```

if the binary has the same name as the package, you don't need to specify it explicitly

Reply:
Thanks for you help!!